media/css/115release.css

```
@@ -0,0 +1,1178 @@
+  --color-purple-80: #6b21a8;
+  --color-purple-90: #581c87;
+
+  --color-magenta-10: #fbe7f9;
+  --color-magenta-20: #f8cff3;
+  --color-magenta-30: #f4a9e8;
+  --color-magenta-40: #ee75d7;
+  --color-magenta-50: #e247c4;
+  --color-magenta-60: #cd26a5;
+  --color-magenta-70: #b01a86;
+  --color-magenta-80: #91186e;
+  --color-magenta-90: #79195c;
+
+  --color-brown-10: #f4e9d7;
+  --color-brown-20: #efdfc4;
+  --color-brown-30: #e4cdab;
+  --color-brown-40: #d7bc96;
+  --color-brown-50: #b6986c;
+  --color-brown-60: #96764b;
+  --color-brown-70: #755b38;
+  --color-brown-80: #51412c;
+  --color-brown-90: #47341f;
+
+  --color-gray-05: #fafafa;
+  --color-gray-10: #f4f4f5;
+  --color-gray-20: #e4e4e7;
+  --color-gray-30: #d4d4d8;
+  --color-gray-40: #a1a1aa;
+  --color-gray-50: #71717a;
+  --color-gray-60: #52525b;
+  --color-gray-70: #3f3f46;
+  --color-gray-80: #27272a;
+  --color-gray-90: #18181b;
+
+  --color-ink-10: #f1f3fa;
+  --color-ink-20: #e3e5f2;
+  --color-ink-30: #cdd0e5;
+  --color-ink-40: #9b9ec2;
+  --color-ink-50: #6e6f9b;
+  --color-ink-60: #52507c;
+  --color-ink-70: #3e3c67;
+  --color-ink-80: #2a284b;
+  --color-ink-90: #1a1838;
+
+  --color-white: #ffffff;
+  --color-black: #000000;
+
+  --bg: #31293e;
+  --txt: var(--color-gray-05);
+
+  --max-width: 1280px;
+  --slider: 50%;
+
+  --icon-glow: var(--color-gray-50);
+  --info-header: var(--color-gray-20);
+  --info-txt: var(--color-gray-10);
+
+  --button-background-color: var(--color-gray-20);
+  --button-hover-text-color: var(--color-gray-70);
+  --button-hover-background-color: var(--color-gray-10);
+  --button-active-background-color: var(--color-gray-30);
+  --button-border-color: var(--color-gray-40);
+  --button-border-radius: 6px;
+  --button-border-size: 2px;
+  --button-text-color: var(--color-gray-70);
+  --button-margin: 9px;
+  --button-padding: 9px;
+  --focus-outline-color: var(--color-blue-50);
+  --focus-outline-offset: 4px;
+  --focus-outline: 4px solid var(--focus-outline-color);
+  --icon-size: 32px;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  font-size: 1rem;
+  background-color: var(--bg);
+  color: var(--txt);
+}
+
+header {
+  --max-width: 70ch;
+  background-image: url("/media/img/thunderbird/backgrounds/blob.png");
+  background-size: min(150%, 150vh);
+  background-position: bottom;
+  background-repeat: no-repeat;
+  padding-inline: 16px;
+  padding-block: 100px 500px;
+}
+
+header h1 {
+  font-weight: 300;
+  font-size: 2.5rem;
+  text-shadow: 0 2px 6px var(--color-ink-90);
+}
+
+.tb-title {
+  font-weight: 600;
+  text-transform: uppercase;
+  font-size: 4rem;
+}
+
+.version,
+.alive {
+  color: transparent;
+  font-weight: 900;
+  line-height: 0;
+  text-shadow: none;
+}
+
+.version {
+  font-size: 10rem;
+  background: linear-gradient(30deg, transparent 25%, var(--color-purple-20));
+  -webkit-background-clip: text;
+  background-clip: text;
+  margin-inline-start: -10rem;
+  margin-block-start: -12rem;
+  vertical-align: super;
```

If we use flex column, we can do this here
```suggestion
  line-height: 1;
  align-self: end;
  margin-block-start: -9rem;
```